prince-of-versions/src/main/java/co/infinum/princeofversions/common/VersionContext.java

```diff
@@ -49,22 +50,28 @@
     private Map<String, String> metadata;
 
     /**
+     * Former minSdk value
+     */
+    private int minVersionMinSdk;
```

Comment:
:) Ok, we definitely need to change the java doc. Also, we should consider changing the variable name to _minimunVersionMinSdk_.